### ISSUE TITLE
Fedprox support

### ISSUE THREAD

**Issue by AFKD98** [IBM/federated-learning-lib#75]
Hi, the aggregator config files generated for fedprox uses the iter average fusion handler. Is the Fedprox capability in the library incomplete or is it supposed to use the iter average fusion handler? 
![image](https://user-images.githubusercontent.com/49161231/129906192-48fefd6c-023a-4948-8582-650169f2c113.png)

**Comment by Yi-Zoey** [comment]
Hi @AFKD98 thanks for using IBM FL. The aggregator config looks good. Note that FedProx modifies the parties' local training process, in particular, it utilizes a different optimizer: Perturbed Gradient Descent. Therefore, you can see in the model config generation ([here](https://github.com/IBM/federated-learning-lib/blob/main/examples/fedprox/generate_configs.py#L83)) it defines the model in a slightly different manner.

**Comment by AFKD98** [comment]
Thank you! That helps.
